src/test/scala/chiselTests/SwitchSpec.scala

```diff
@@ -32,4 +32,22 @@ class SwitchSpec extends ChiselFlatSpec with Utils {
       })
     }
   }
+  it should "provide useful source locators" in {
+    val verilog = ChiselStage.emitVerilog(new Module {
+      val io = IO(new Bundle {
+        val in = Input(UInt(2.W))
+        val out = Output(UInt(2.W))
+      })
+
+      io.out := 0.U
+      switch (io.in) {
+        is (0.U) { io.out := 3.U }
+        is (1.U) { io.out := 0.U }
+        is (2.U) { io.out := 1.U }
+        is (3.U) { io.out := 3.U }
+      }
+    })
+
+    assert(!verilog.contains("Conditional.scala"))
```

Comment:
Nit: Scalatest has great support for this type of test without resorting to `assert`. You should be able to express this with:

```suggestion
    verilog should not contain ("Conditional.scala")
```

Reply:
I tried that and it did not work correctly. Apparently, for strings you have to do `string should not include "substring"` instead.

Reply:
:100: